cvat-ui/src/components/annotation-page/standard-workspace/controls-side-bar/handle-popover-visibility.tsx

```diff
@@ -23,19 +23,15 @@ export default function withVisibilityHandling(WrappedComponent: typeof Popover,
                 {...rest}
                 trigger={visible ? 'click' : 'hover'}
                 overlayClassName={overlayClassNames.join(' ').trim()}
+                afterVisibleChange={(_visible: boolean) => {
```

Comment:
We use ``afterVisibleChange``, but it is not a part of antd API, so it can be changed in the future.
What is more, as I found, afterVisibleChange does not guarantee that the popover does not have ``pointer-events: none`` and it is triggered several times. .
So, maybe current implementation with (waitFor) is better.

Reply:
Returned.